tutorials/metrics.md

```diff
@@ -28,18 +28,20 @@ endpoint manually via [curl][1].
+| `crio_operations_errors`                     | every CRI-O RPC\*                                                                                                                       | Counter | Cumulative number of CRI-O operation errors by operation type.                                       |
+| `crio_image_pulls_by_digest`                 | `name`, `digest`, `mediatype`, `size`                                                                                                   | Counter | Bytes transferred by CRI-O image pulls by digest.                                                    |
+| `crio_image_pulls_by_name`                   | `name`, `size`                                                                                                                          | Counter | Bytes transferred by CRI-O image pulls by name.                                                      |
+| `crio_image_pulls_by_name_skipped`           | `name`                                                                                                                                  | Counter | Bytes skipped by CRI-O image pulls by name.                                                          |
+| `crio_image_pulls_successes`                 | `name`                                                                                                                                  | Counter | Successful image pulls by image name                                                                 |
+| `crio_image_pulls_failures`                  | `name`, `error`                                                                                                                         | Counter | Failed image pulls by image name and their error category.                                           |
+| `crio_image_layer_reuse`                     | `name`                                                                                                                                  | Counter | Reused (not pulled) local image layer count by name.                                                 |
+| `crio_containers_oom_total`                  |                                                                                                                                         | Counter | Total amount of containers killed because they ran out of memory (OOM)                               |
+| `crio_containers_oom`                        | `name`                                                                                                                                  | Counter | Containers killed because they ran out of memory (OOM) by their name                                 |
```

Comment:
Maybe it's late in the day brain, but I'm not gronking the difference between these last two....

Reply:
One is the total amount of OOM kills and the second is the oom kills by their container name :upside_down_face: